fitlib/changes.py

```diff
@@ -163,7 +164,7 @@ def getChangedItems(fitTrackedData, trackedItems=None, paths=None, pathArgs=None
 
     if paths != None:
         if len(paths) == 0:
-            return ({}, set(), set(), set(), set(), {})
+            return ({}, set(), set(), set(), set(), {}, {}) # return all 7 as empty
```

Comment:
If all seven are not returned, a syntax error will be caused in some use cases. For example, this will fail:
> git-fit abcd